tools/metilene/.shed.yml

```diff
@@ -0,0 +1,8 @@
+categories:
+- RNA
+name: metilene
```

Comment:
Can you please add the other fields as well. Like this one? https://github.com/galaxyproject/tools-iuc/blob/master/tools/deseq2/.shed.yml